src/nix-daemon/nix-daemon.cc

```diff
@@ -573,8 +573,13 @@ static void performOp(TunnelLogger * logger, ref<Store> store,
                     ;
                 else if (setSubstituters(settings.extraSubstituters))
                     ;
-                else
+                else {
+                    logger->log(lvlInfo,
+                        "WARNING: ignoring the " + name + " setting. " +
+                        "You need to declare your unix user as a nix.trustedUser " +
+                        "to override this setting.");
```

Comment:
What would you think about:

> Warning:  «name» is a restricted setting and you are not a trusted user.

A few things related:

lvlInfo is not really a Warning :p 
and `nix.trustedUser` is a NixOS setting, not a nix.conf setting.

Reply:
Better indeed :)

> lvlInfo is not really a Warning :p

Totally agree! That said, I did not find anything better:

```
typedef enum {
    lvlError = 0,
    lvlInfo,
    lvlTalkative,
    lvlChatty,
    lvlDebug,
    lvlVomit
} Verbosity;
```

Do you think lvlError would be better suited here?